manifests/k/Kakao/KakaoTalk/3.2.9.2815/Kakao.KakaoTalk.locale.ko-KR.yaml

```diff
@@ -3,7 +3,11 @@ PackageIdentifier: Kakao.KakaoTalk
 PackageVersion: 3.2.9.2815
 PackageLocale: ko-KR
 Publisher: Kakao Corp.
+PublisherUrl: https://www.kakaocorp.com/
+PublisherSupportUrl: https://cs.kakao.com/
+PrivacyUrl: https://www.kakao.com/policy/privacy/
 PackageName: KakaoTalk
+PackageUrl: https://www.kakaocorp.com/page/service/service/KakaoTalk/
 License: Copyright © Kakao Corp. All rights reserved.
 ShortDescription: KakaoTalk Setup

```

Comment:
`ShortDescription: KakaoTalk Setup`

Could you please write a description as well 👍

Reply:
fixed :)